### PR TITLE
[#2] Integrate react-native-fitness, refactor screen navigation

### PR DESCRIPTION
**Pull request by francisli**
WIP, do not merge.

This is a very quick, hard-coded test of the react-native-fitness library. I can use my personal developer credentials to build the app, run it on my iPhone, and see daily step counts displayed in console output. I have not tested on Android yet.

@ericouyang I'm going to open a new separate issue for you to set up our Google and Apple developer teams. We should get those configured and set up before anyone goes any further- the reason is, accessing health related info requires setting up unique app ids and requesting entitlements from Apple and Google- if we do this on our personal accounts, it will be a pain in the @$$ to transfer those to the organizational accounts later (without renaming/re-issuing new ids and entitlements).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,31 +33,25 @@
...
 
+      For Mac OS X, configure your shell environment to reference the newly installed
+      developer tools. In the standard Mac OS Terminal, edit (or create, if needed)
+      your .bash_profile in your home directory and add the following (assuming
+      default installation locations):
 
       ```
-      brew tap AdoptOpenJDK/openjdk
-      brew cask install adoptopenjdk8
+      export JAVA_HOME=/Applications/Android\ Studio.app/Contents/jre/jdk/Contents/Home/
```
Comment (francisli) [inline README.md]: @ericouyang does this alternative Android setup work for you?

Comment (francisli) [reply README.md]: I think I finally found the Android setup instructions you followed on the React Native site, will update to include instructions on how to find it... I think they're a little bit out of date (i.e. ANDROID_HOME is technically a deprecated env var, now replaced with ANDROID_SDK_ROOT...)...

**Comment by francisli** [comment]
@johnsonphan95 can you let me know if the changes in this PR look OK for you?

So, in order to test the fitness library integration, I needed to have an opportune place in the UI flow to request appropriate permissions. In this case, I've tied it to the "START" button on the last screen in the onboarding flow. If permitted, the onboarding flow will then dismiss to a placeholder "main" screen which will attempt to query and display the user's current day's steps.

**Comment by francisli** [comment]
Right now, I can confirm that the library is working, and I can request permissions for fitness data.

On iOS, I can see my daily steps on my personal iPhone.

On Android, via console logging, I can confirm that our app successfully "subscribes" to the fitness data (an Android-only specific step).

However, on my test devices, so far I have not yet been able to get any real step count data returned- it always returns an empty response.

**Comment by francisli** [comment]
@johnsonphan95 as part of the screen re-factor, I've upgraded react-navigation to the latest version, and changed the code style to match the newer conventions seen in the latest version of the docs.

**Comment by johnsonphan95** [comment]
@francisli Everything looks in order and much cleaner, thanks for that! Looks good to me

**Pull request merged**